lib/frisby.js

```diff
@@ -1208,8 +1208,8 @@ function _jsonContains(jsonBody, jsonTest) {
 
     // Ensure property exists
     if(jsonBody) {
-      var kt = typeof jsonTest[key];
-      if(kt === "object") {
+      var kt = _toType(jsonTest[key]);
+      if(kt === "object" || kt === "array") {
```

Comment:
The array part is pretty tricky. Are we checking all elements against the same matcher or do we support having more then one matcher in the array? I believe to keep code simple it's better to avoid arrays.